scripts/install_tethys.sh

```diff
@@ -530,9 +522,9 @@ then
 
     echo "export NGINX_USER='${NGINX_USER}'" >> "${ACTIVATE_SCRIPT}"
     echo "export NGINX_HOME='${NGINX_HOME}'" >> "${ACTIVATE_SCRIPT}"
-    echo "alias tethys_user_own='sudo chown -R \${USER} \"\${TETHYS_SRC}\" \"\${TETHYS_HOME}/static\" \"\${TETHYS_HOME}/workspaces\" \"\${TETHYS_HOME}/apps\"'" >> "${ACTIVATE_SCRIPT}"
+    echo "alias tethys_user_own='sudo chown -R \${USER} \"${TETHYS_SRC}\" \"${TETHYS_HOME}/static\" \"${TETHYS_HOME}/workspaces\" \"${TETHYS_HOME}/apps\"'" >> "${ACTIVATE_SCRIPT}"
```

Comment:
Are ``TETHYS_SRC`` and ``TETHYS_HOME`` defined in this script still?

Reply:
The SERVIR folks rely pretty heavily on these aliases for managing their manual deploys.

Reply:
Yes, both `TETHYS_SRC` and `TETHYS_HOME` are defined in the install script, but I have removed them from the activate script. The changes here have the effect of substituting the values for those variables when the activate scripts are written, rather than writing the variable names in the script so they are substituted at runtime.